src/applications/lgy/coe/containers/App.jsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react';
-
+import Breadcrumbs from '@department-of-veterans-affairs/component-library/Breadcrumbs';
 import RoutedSavableApp from 'platform/forms/save-in-progress/RoutedSavableApp';
 import formConfig from '../config/form';
 
 export default function App({ location, children }) {
   return (
-    <RoutedSavableApp formConfig={formConfig} currentLocation={location}>
-      {children}
-    </RoutedSavableApp>
+    <>
+      <Breadcrumbs>
```

Comment:
Not sure if one way is better than the other, but you can also add breadcrumbs via `registry.json` using the `includeBreadcrumbs` and `breadcrumbs_override` parameters.

Reply:
@micahchiang Yep, I moved them to the registry file, I was just putting them in the App file while testing since you don't have to run the build to see them there.